packages/realm-react/README.md

```diff
@@ -6,7 +6,7 @@
   Realm React
 </h1>
 
-A better way to use Realm with React Native applications.
+Local Data Made Simple.
```

Comment:
Is this an official Realm thing? "Local" sounds a bit odd to me as one of the key selling points is sync

Reply:
It's not an official thing.  If you have any ideas of what to change that to, i'm all about it

Reply:
Can we go with the official one: "Build better apps, faster"?

Reply:
Sounds good!